README.md

```diff
@@ -228,6 +228,8 @@ $ pip install tern scancode-toolkit
 $ tern report -x scancode -i golang:1.12-alpine
 ```
 
+If you are running Scancode for the first time, this will take a long time to complete. Once completed, subsequent runs will be much faster as the data will be cached for future use.
```

Comment:
Do you have an estimate of what "a long time to complete" is?

Reply:
It depends on the container image. I've had to run scancode for a couple of hours or even overnight for some containers.